src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -355,6 +362,25 @@ internal class RecommendationService(
     }
   }
 
+  private fun sendRecommendationStartedEvent(recommendationId: Long) {
+    val crn = recommendationRepository.findById(recommendationId).map { it.data.crn }.get()
+    val payload = MrdEvent(
+      message = MrdEventMessageBody(
+        eventType = "prison-recall.recommendation.started",
+        version = 1,
+        description = "Recommendation started (recall or no recall)",
+        occurredAt = LocalDateTime.now(),
+        detailUrl = "", // TODO TBD
+        personReference = PersonReference(listOf(TypeValue(type = "CRN", value = crn))),
+        additionalInformation = AdditionalInformation(recommendationUrl = "http://mrd.case.crn/overview")
```

Comment:
That was just an example URL for the ticket - it needs to be the correct URL to the overview tab for the case that has had the recommendation started.

Reply:
hardcoded to prod for now, but I have asked Matt R for some help on making it dynamic to detect the host domain of whichever environment is in use